server/routes/index.js

```diff
@@ -1399,13 +1350,15 @@ router
     try {
       const { appDb } = req.app.locals
       const { user } = req
-      const report = await appDb.collection('reports').findOne({
-        _id: ObjectID(req.params.id),
-        $or: [{ user }, { readers: user }],
+      const report = await appDb.reports.findUniqueOrThrow({
+        where: {
+          _id: ObjectId(req.params.id),
+          OR: [{ user }, { readers: user }],
```

Comment:
does `{ readers: user }` mean that it will find reports where the user is included in the array of readers? We don't need to specify any sort of `IN` syntax to make that work?